index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <meta charset='utf-8'>
+   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+   <title>My Portfolio</title>
+   <meta name='viewport' content='width=device-width, initial-scale=1'>
+   <link rel='stylesheet' type='text/css' media='screen' href='styles.css'>
+   <script src='main.js'></script>
+</head>
+<body>
+   <header>
+      <div class="logo">
+         <h3>My Logo</h3>
+      </div>
+      <div class="hamburger">
+         <span></span>
+         <span></span>
+         <span></span>
+      </div>
+   </header>
+   <main>
+      <section id="headline">
+         <h1>I’m Mariane Glad to see you!</h1>
+         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+         <h4>Let's connect</h4>
+         <div class="socials">
+            <img src="images/socials/Vector.svg"/>
+            <img src="images/socials/linkedin.svg"/>
+            <img src="images/socials/Vector (2).svg"/>
+            <img src="images/socials/Vector (3).svg"/>
+            <img src="images/socials/Vector (4).svg"/>
+         </div>
```

Comment:
- [ ] For the social media icons, kindly made the following changes:

- The `div` tag with the class `socials` should be a `ul` tag. 
- Each image should be wrapped in a `li` tag and then `a` tag. Wrapping icons in `a` tag is not only good for semantic reasons but it maps users to your social media page through the `href` attribute of the `a` tag.

So the section should be like this:
```HTML
<ul>
  <li>
    <a href="twitter.com/username"><img src="images/socials/Vector.svg"/></a>
  </li>
...
</ul>
```

Reply:
Thanks so much for this. I will implement asap